docs/DeveloperGuide.md

```diff
@@ -422,12 +420,13 @@ Use case ends.
 
 2a. The list is empty
 
-> Use case ends
+ Use case ends
 
 3a. The given index is invalid
 
-> 3a1. DoTomorrow shows an error message <br>
-  Use case resumes at step 2
+> 3a1. DoTomorrow shows an error message
+
```

Comment:
Is there a reason you're adding a free line rather than the `<br>`? This doesn't actually render to the same thing. There will be more spacing between the lined this way. If you don't like the `<br>` tag, you can end a line with multiple spaces to do the same thing. But we seem to be using `<br>` tags throughout.